wiki/Reporting_Bad_Behaviour/id.md

```diff
@@ -98,8 +103,8 @@ Sebagian besar masalah pada pesan pribadi seperti penghinaan tidak akan ditangan
 
 Pengguna dapat menggunakan sistem abaikan untuk menghindari pesan apa pun yang dikirim oleh pengguna termasuk pesan di channel.
 
-**Catatan:** Daftar Abaikan akan disetel ulang saat anda menutup klien game.
+**Catatan:** Daftar abaikan akan disetel ulang saat Anda menutup klien gim.
```

Comment:
```suggestion
**Catatan:** Daftar Abaikan akan disetel ulang saat Anda menutup klien gim.
```
Yang 'Abaikan' ini engga pake kapital ? Mungkin bisa diganti ke kapital karena "Ignore" di aslinya juga huruf kapital , kurang yakin juga sebenarnya karena mungkin huruf kapital cuma di implementasikan buat kata pertama. Tapi gpp kutulis aja itung2 nambah review awkwk

Reply:
![image](https://user-images.githubusercontent.com/69805397/107665446-0480c280-6cc0-11eb-85b7-9715d994d451.png)
diganti jadi `Daftar pengabaian` sesuai sama yang ada di in-game.